kerosene/config/trainers.py

```diff
@@ -145,9 +161,9 @@ def metric_params(self):
     @classmethod
     def from_dict(cls, model_name, config_dict):
         return cls(model_name, config_dict["type"], config_dict["params"], config_dict["optimizer"]["type"],
-                   config_dict["optimizer"]["params"], config_dict["scheduler"]["type"],
-                   config_dict["scheduler"]["params"], config_dict["criterion"]["type"],
-                   config_dict["criterion"]["params"], config_dict["metric"]["type"],
+                   config_dict["optimizer"]["max_grad_norm"], config_dict["optimizer"]["params"],
```

Comment:
Moved one step above in dict hierarchy. Would now be: 

models:
  Segmenter:
    name: "UNet3D"
    type: "UNet3D"
    params:
      ...
    optimizer:
      type: "FusedSGD"
      params:
        ...
    scheduler:
      type: "ReduceLROnPlateau"
      params:
        ...
    criterion:
      ...
    metric:
      ...
    params:
        max_grad_norm: 1.0

Reply:
Can't put it anywhere else I think. Cannot be in 
models:
   type:...
   name:...
   params:
      max_grad_norm

because these params are passed directly to class initializer of the model. 

Could be in the params of the optimizer, but again these params are directly passed when instantiating the optimizer.

Reply:
Created a section "gradients" in config file.